### DISCUSSION TITLE
Problem with firmware update (0X800701B1)

### DISCUSSION THREAD

**Discussion by sliva95** [arkypita/LaserGRBL#1615] [Q&A]
Hi,

i'm new one here. Please, can you help me with update firmware on Ortur master 2?
After i open folder from Ortur, i put new firmware (OLM_2_PRO_Firmware_Release_1.52) to folder and shows me this error: 0X800701B1.

I try update windows 10, i try other usb ports, i try Ccleaner for upgrade all drivers and problem is still here...

Thank you for answers.

**Comment by arkypita** [comment]
When I updated mine I had no problems, I simply followed the directions found in the zip file you download from here: https://ortur.tech/firmware/OLM_Series/OLM_2_Firmware_Release_1.4.zip

In any case this is not the right place to ask for help, you would certainly find more competent people on the Ortur facebook page or by contacting them at their assistance service. 

https://www.facebook.com/groups/orturusergroup
https://ortur.tech/support/
